applications/app/controllers/AtomPageController.scala

```diff
@@ -96,12 +96,15 @@ class AtomPageController(
       )
     }
 
+  def renderNoJs(atomType: String, id: String): Action[AnyContent] = render(atomType, id, false, false)
+
+  def renderNoJsVerticalScroll(atomType: String, id: String): Action[AnyContent] = render(atomType, id, false, true)
+
   def render(
       atomType: String,
       id: String,
       isJsEnabled: Boolean,
       hasVerticalScrollbar: Boolean,
-      inApp: Boolean,
```

Comment:
This parameter was introduced in November 2017 with https://github.com/guardian/frontend/pull/18213, and became obsolete in October 2019 [with](https://github.com/guardian/frontend/pull/21897#discussion_r941555632) https://github.com/guardian/frontend/pull/21897.